components/eam/src/physics/cam/ndrop.F90

```diff
@@ -524,10 +484,10 @@ subroutine dropmixnuc( &
       csbot(pver) = cs(icol,pver)
       csbot_cscen(pver) = 1.0_r8
 
-!  Initialize 1D (in space) versions of interstitial and airborne aerosol
+!  Initialize 1D (in space) versions of interstitial and cloud borne aerosol
 
       nsav = 1
-      nnew = 2
+
```

Comment:
Actually, these are not constants.  Subroutine 'update_from_explmix' (as did the previous code) stores two time levels of aerosol fields, denoted by a length-two array dimension whose indices are 'nsav' and 'nnew'.   For each subloop in update_from_explmix, new values are put into the nnew element, while the old values are sent to the nsav element.  However, this is not done by physically copying the arrays, but by switching the values of 'nsav' and 'nnew' for each subloop.  This is also why these values are passed as arguments into 'update_from_explmix' -- because there did not seem to be a straightforward way of determining what the values of 'nsav' and 'nnew'  would be once all subloops are completed.